src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,9 +5,28 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
+    private int width;
+    private String series;
```

Comment:
Don't forget about access modifiers for your custom fields in `Truck` class. Consider using protected access modifier if subclassing is intended, otherwise keep it private.